workers_test.go

```diff
@@ -179,15 +179,38 @@ const (
   "errors": [],
   "messages": []
 }`
+	workerScript                  = `addEventListener('fetch', event => {
+    event.passThroughOnException()
+event.respondWith(handleRequest(event.request))
+})
+
+async function handleRequest(request) {
+    return fetch(request)
+}`
+	workerModuleScript            = `export default {
+    async fetch(request, env, event) {
+     event.passThroughOnException()
+    return fetch(request)
+    }
+}`
+	workerModuleDownloadResponse = `
+--workermoduledownloadresponse
+Content-Disposition: form-data; name="worker.js"
+
+export default {
+    async fetch(request, env, event) {
+     event.passThroughOnException()
+    return fetch(request)
+    }
+}
+--workermoduledownloadresponse--
+`
 )
 
 var (
 	successResponse               = Response{Success: true, Errors: []ResponseInfo{}, Messages: []ResponseInfo{}}
-	workerScript                  = "addEventListener('fetch', event => {\n    event.passThroughOnException()\nevent.respondWith(handleRequest(event.request))\n})\n\nasync function handleRequest(request) {\n    return fetch(request)\n}"
```

Comment:
Way easier to read using backticks than embedded newlines, IMO. No functional change here.